charts/velero/values.yaml

```diff
@@ -72,6 +72,10 @@ metrics:
 # Install CRDs as a templates. Enabled by default.
 installCRDs: true
 
+# Enable/disable helm hooks annotations for CRDs (Helm v2) and templates
+# You should disable this if using a deploy tool that doesn't support helm hooks
```

Comment:
1. I don't think it's for helm2 only, these helm hooks also available on helm3
2. maybe we could mention the deploy tool ArgoCD to help people easier to understand when to disable helm hooks annotations.

Reply:
For CRDs it was helm2 only, because in helm3 CRDs can't be templated, but for all else it was for both v2 and v3.

Reply:
> For CRDs it was helm2 only, because in helm3 CRDs can't be templated, but for all else it was for both v2 and v3.

```
helm template vmware-tanzu/velero --include-crds
```

Reply:
Hooks are available for Helm3, but in this chart the option enableHelmHooks will only enable/disable helm hooks for helm2 CRDs (templates/crds.yaml) and for all other templates on both versions. 
I've made the changes in the comments and chart version.